manifest.json

```diff
@@ -6,7 +6,7 @@
   "version": "1.0.1",
   "author": "Dara Javaherian",
     
-  "permissions": ["tabs", "http://*/*", "https://*/*"],
+  "permissions": ["tabs", "*://*.youtube.com/*"],
```

Comment:
Will this work for simply `youtube.com`? Also, would we need to cover `youtu.be` etc?